internal/pkg/template/templates/environment/partials/elb-access-logs.yml

```diff
@@ -0,0 +1,38 @@
+ELBAccessLogsBucketPolicy:
```

Comment:
How does this work? It seems like we don't refer to `ELBAccessLogsBucketPolicy` anywhere. We just need this policy defined here and then ELB will be able to write to the bucket?

Reply:
No no, if you see [here](https://github.com/aws/copilot-cli/pull/3849/files/8b5ae7abb677ddcaf6c8067697ae395c9b9ebf2b#diff-3622335dad872c1b6514d7b8e98203d3c30f9ab68350d949b14ac3b9a6774dfeR6) we are setting `Bucket` field on the policy which tells the policy that it needs to be applied on that particular bucket. Does that make sense?

Reply:
I see `AWS::S3::BucketPolicy` is a special one. Thanks!